### PR TITLE
scripts/install.in: add riscv64 support to installer

### PR DESCRIPTION
**Pull request by jdek**
# Motivation

The artifacts are already built and hosted, the install script just needs to be taught about riscv64.

# Context

Making this change manually to the install script the Nix installation was able to succeed. 

# Priorities and Process

Add :+1: to [pull requests you find important](https://github.com/NixOS/nix/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc).

The Nix maintainer team uses a [GitHub project board](https://github.com/orgs/NixOS/projects/19) to [schedule and track reviews](https://github.com/NixOS/nix/tree/master/maintainers#project-board-protocol).

### CODE REVIEW

**Comment by cole-h** [comment]
I believe you'll also need to add riscv64 here: https://github.com/NixOS/nix/blob/ef5c846e257e1e284ad47ed6be4308d190fe6531/maintainers/upload-release.pl#L161-L165

**Comment by cole-h** [comment]
Oh oops you're right, I meant to point at the `fallback-paths.nix` writing. Not sure why I decided the place I pointed to was the right place :sweat_smile:

**Review by roberth — APPROVED** [review]
Looks to be complete.
Not sure about fallback paths, but I don't see harm in it, so :+1: from me.

**Comment by edolstra** [comment]
Merging this with the caveat that it's best-effort (since we're not actually testing whether it works, only cross-building) and may be disabled if it breaks and blocks the release (since none of the maintainers are really capable of fixing riscv64 issues).

**Comment by Mic92** [comment]
I also got access to a bigger riscv64 machine.

**Pull request merged**
